### ISSUE TITLE
Ecmarkup no longer runs after updating grammarkdown to 2.1.0

### ISSUE THREAD

**Issue by bakkot** [rbuckton/grammarkdown#47]
Ecmarkup no longer runs after updating grammarkdown to 2.1.0 (or 2.1.1):

```
$ ./bin/ecmarkup.js ../ecma262/spec.html out.html
Cannot add property _signal, object is not extensible
    at CancellationToken.[@esfx/cancelable:Cancelable.cancelSignal] (/Code/ecmarkup/node_modules/prex/out/lib/cancellation.js:302:26)
    at Function.from (/Code/ecmarkup/node_modules/@esfx/async-canceltoken/dist/index.js:207:72)
    at Object.toCancelToken (/Code/ecmarkup/node_modules/grammarkdown/dist/core.js:296:48)
    at Function.convert (/Code/ecmarkup/node_modules/grammarkdown/dist/grammar.js:81:36)
    at Function.enter (/Code/ecmarkup/lib/Grammar.js:60:49)
    at walk (/Code/ecmarkup/lib/Spec.js:634:17)
    at walk (/Code/ecmarkup/lib/Spec.js:638:13)
    at walk (/Code/ecmarkup/lib/Spec.js:638:13)
    at walk (/Code/ecmarkup/lib/Spec.js:638:13)
    at walk (/Code/ecmarkup/lib/Spec.js:638:13)
```

I am guessing this has to do with https://github.com/rbuckton/grammarkdown/pull/44.

**Comment by rbuckton** [comment]
It's a bug in prex@0.4.6. I'll look into it tomorrow.

**Comment by rbuckton** [comment]
I fixed the issue in `prex@0.4.7` and updated the minimum dependency in grammarkdown. This should be fixed as of `grammarkdown@2.1.2`

**Comment by bakkot** [comment]
Yup, ecmarkup is successfully updated to `grammarkdown@2.1.2`. Thanks!
